stripe/src/test/java/com/stripe/android/view/PaymentUtilsTest.java

```diff
@@ -0,0 +1,79 @@
+package com.stripe.android.view;
+
+import android.support.v7.app.AppCompatActivity;
+
+import com.stripe.android.BuildConfig;
+import com.stripe.android.R;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.android.controller.ActivityController;
+import org.robolectric.annotation.Config;
+
+import java.util.Currency;
+import java.util.Locale;
+
+import static junit.framework.Assert.assertEquals;
+
+/**
+ * Test class for {@link PaymentUtils}
+ */
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = 25, constants = BuildConfig.class)
+public class PaymentUtilsTest {
+
+    ActivityController<AppCompatActivity> mActivityController;
+
+    @Before
+    public void setup() {
+        Locale.setDefault(Locale.US);
+        mActivityController = Robolectric.buildActivity(AppCompatActivity.class).create().start();
+    }
+
+    @Test
+    public void zeroPriceStringTest() {
```

Comment:
I hadn't been strict about it, but I think it helps particularly in more complicated test situations to kind of stick to a format of `methodName_whenCondition_doesThing`. There are always places where this doesn't make sense (like just initializing a view and making sure all the stuff shows up), but I think this might be applicable here.